convert2rhel/data/7/ppc64/configs/centos-7-ppc64.cfg

```diff
@@ -10,11 +10,13 @@ gpg_fingerprints =
 # List of packages to be removed before the system conversion starts.
 # Delimited by any whitespace(s).
 excluded_pkgs =
-  centos-bookmarks
-  centos-logos
-  centos-indexhtml
-  libreport-centos
-  libreport-plugin-mantisbt
```

Comment:
What happened to this plugin? Do we have a reason to remove it from the exclude list?